java/libphonenumber/src/com/google/i18n/phonenumbers/nano/Phonemetadata.java

```diff
@@ -207,6 +207,12 @@ public static NumberFormat parseFrom(
     // optional string possible_number_pattern = 3;
     public java.lang.String possibleNumberPattern;
 
+    // repeated int32 possible_length = 9;
```

Comment:
Nice, did you generate this following [these instructions](https://github.com/googlei18n/libphonenumber/blob/master/java/lib/protobuf-javanano-readme.md)? I'm excited to hear they worked for someone other than me :)?